packages/perseus/src/__tests__/renderer.test.tsx

```diff
@@ -1605,35 +1606,36 @@ describe("renderer", () => {
         it("should return all empty widgets", async () => {
             // Arrange
             const {renderer} = renderQuestion({
-                ...question2,
+                ...question3,
                 content:
-                    "Input 1: [[☃ numeric-input 1]]\n\n" +
-                    "Input 2: [[☃ numeric-input 2]]",
+                    "Input 1: [[☃ expression 1]]\n\n" +
+                    "Input 2: [[☃ expression 2]]",
                 widgets: {
-                    ...question2.widgets,
-                    "numeric-input 2": question2.widgets["numeric-input 1"],
+                    ...question3.widgets,
+                    "expression 2": question3.widgets["expression 1"],
                 },
             });
             await userEvent.type(screen.getAllByRole("textbox")[0], "150");
+            act(() => jest.runOnlyPendingTimers());
```

Comment:
This is because we are now using the `expression` widget. I couldn't figure out why the `userEvent.type()` call above wasn't working. In other places of Perseus where we test `expression` we also do this. I _think_ the reason is that it takes a render cycle for the typing to show up in the expression widget. `expression` is based on MathInput which uses MathQuill under the hood. Probably related to that.